src/app/map/map.component.ts

```diff
@@ -19,7 +19,9 @@ import { StylesService, StyleRule } from '../services/styles.service';
 import { GeoJsonLayer } from '@deck.gl/layers';
 import { GoogleMapsOverlay } from '@deck.gl/google-maps';
 import bbox from '@turf/bbox';
-import { GeoJSONService, GeoJSONFeature } from '../services/geojson.service';
+import { GeoJSONService } from '../services/geojson.service';
+import {Feature} from 'geojson';
```

Comment:
Nit: `s/{Feature}/{ Feature }/`

Running `ng lint fix` should fix this.

Reply:
Yes, I did not pay much attention tonformattingthanksnfor nitpicking!